kong/api/routes/debug.lua

```diff
@@ -1,16 +1,20 @@
-local get_sys_filter_level = require("ngx.errlog").get_sys_filter_level
-local set_log_level = require("resty.kong.log").set_log_level
+local set_log_level               = require("resty.kong.log").set_log_level
+local cjson                       = require("cjson.safe")
 
-local LOG_LEVELS = require("kong.constants").LOG_LEVELS
+local LOG_LEVELS                  = require("kong.constants").LOG_LEVELS
 
-local ngx = ngx
-local kong = kong
-local pcall = pcall
-local type = type
-local tostring = tostring
+local ngx                         = ngx
+local kong                        = kong
+local pcall                       = pcall
+local type                        = type
+local tostring                    = tostring
+
+local get_log_level               = require("resty.kong.log").get_log_level
+
+local NODE_LEVEL_BROADCAST        = false
+local CLUSTER_LEVEL_BROADCAST     = true
+local DEFAULT_LOG_LEVEL_TIMEOUT   = 60 * 10 -- 10 minutes
```

Comment:
suggest adding unit into the variable name, like `DEFAULT_LOG_LEVEL_TIMEOUT_SECONDS`